src/Components/Analyzers/test/ComponentCaptureExtraAttributesParameterMustBeUniqueTest.cs

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
+using TestHelper;
+using Xunit;
+
+namespace Microsoft.AspNetCore.Components.Analyzers.Test
+{
+    public class ComponentCaptureExtraAttributesParameterMustBeUniqueTest : DiagnosticVerifier
+    {
+        [Fact]
+        public void IgnoresPropertiesWithCaptureExtraAttributesFalse()
+        {
+            var test = $@"
+    namespace ConsoleApplication1
+    {{
+        using System.Collections.Generic;
+        using {typeof(ParameterAttribute).Namespace};
+        class TypeName
+        {{
+            [Parameter(CaptureExtraAttributes = false)] string MyProperty {{ get; set; }}
```

Comment:
It will work in the correct way. Processing of the diagnostics is separate. We process the property types per-property, then we process the uniqueness per-type.